src/references/damageable_objects.md

```diff
@@ -0,0 +1,93 @@
+---
+id: damageable_reference
+name: Damageable Objects Reference
+title: Damageable Objects Reference
+tags:
+    - Reference
+---
+
+# Damageable Objects
+
+## Summary
```

Comment:
Summary --> Overview

(I kinda like summary better, but we have a convention and would need to switch it over entirely, so stick to Overview for now)